### ISSUE TITLE
[22414] Tiered approval or Multi consent approval

### ISSUE THREAD

**Issue by Nness** [DeskDirector/UserVoice#455]
Advanced approval system, where tiered approval or ticket that require multiple user to approve.

**Comment by prysmcorp** [comment]
Yes! Often workflows require different sign offs or approvals from varying level of department heads or managerial staff. Huge value add (in my opinion) for those that rack approval workflows.

**Comment by gcao417** [comment]
[573290] Is there a way to force multiple approvers
